tests/command_palette/test_declare_sources.py

```diff
@@ -38,10 +39,8 @@ class AppWithSources(AppWithActiveCommandPalette):
 async def test_app_command_sources() -> None:
     """Command sources declared on an app should be in the command palette."""
     async with AppWithSources().run_test() as pilot:
-        assert (
-            pilot.app.query_one(CommandPalette)._provider_classes
-            == AppWithSources.COMMANDS
-        )
```

Comment:
Does this change mean we can no longer query for the `CommandPalette`?
Don't we find this awkward?

Reply:
I don't think there's ever been a reasonable moment or reason for anyone to need to query for the `CommandPalette`. Probably the only time you'd want to look for it is to check if it's open, and [there's a method for that](https://textual.textualize.io/api/command/#textual.command.CommandPalette.is_open).